pkg/i2gw/providers/istio/converter_test.go

```diff
@@ -1945,3 +1997,41 @@ func Test_converter_generateReferences(t *testing.T) {
 		})
 	}
 }
+
+func Test_convertHostnames(t *testing.T) {
```

Comment:
Can we add a testcase for mix of valid and invalid hostnames? 

Like <ip>,*,<some valid hostname>